src/features/presence/__tests__/__snapshots__/PresenceLink.test.js.snap

```diff
@@ -25,16 +25,15 @@ exports[`features/presence/PresenceLink render() should change the flyoutPositio
   <Overlay
     shouldDefaultFocus={false}
   >
-    <PresenceDropdown
-      className="presence-dropdown"
+    <PresenceCollaboratorsList
       collaborators={
         Array [
           Object {
             "avatarUrl": "",
             "id": "1",
             "interactedAt": 999,
             "interactionType": "user.item_preview",
-            "isActive": true,
+            "isActive": false,
```

Comment:
Is this change expected?

Reply:
Yeah, I refactored to reuse the `collaboratorsList` mock